gcloud/datastore/helpers.py

```diff
@@ -218,44 +218,6 @@ def _get_value_from_property_pb(property_pb):
     return _get_value_from_value_pb(property_pb.value)
 
 
-def _set_protobuf_property(property_pb, name, value, indexed):
```

Comment:
Was this intentionally removed? Was added in #512 and it seems it should be used in `Batch.put()`?